tests/jenkins/TestRunIntegTestScript.groovy

```diff
@@ -82,14 +82,14 @@ class TestRunIntegTestScript extends BuildPipelineTest {
     @Test
     void 'IntegTest LocalPath SwitchNonRoot=false'() {
         runScript("tests/jenkins/jobs/RunIntegTestScript_LocalPath_Jenkinsfile")
-        assertThat(getShellCommands('sh', 'test.sh'), hasItems('  ./test.sh integ-test tests/data/opensearch-1.3.0-test.yml --component OpenSearch --test-run-id null --paths opensearch=tests/jenkins/artifacts/tar '))
+        assertThat(getShellCommands('sh', 'test.sh'), hasItems('env  ./test.sh integ-test tests/data/opensearch-1.3.0-test.yml --component OpenSearch --test-run-id null --paths opensearch=tests/jenkins/artifacts/tar '))
```

Comment:
the env should have java_home right? Its going null here

Reply:
I have a fix now java_home is not always needed, but path can stay so will just add that.

Reply:
Added fix.